tests/cypress/support/commands.ts

```diff
@@ -117,13 +121,17 @@ Cypress.Commands.add('fleetNamespaceToggle', (toggleOption='local') => {
 // Command to delete all rows if check box and delete button are present
 // Note: This function may be substituted by 'cypressLib.deleteAllResources' 
 // when hardcoded texts present can be parameterized
-Cypress.Commands.add('deleteAll', () => {
+Cypress.Commands.add('deleteAll', (fleetCheck=true) => {
   cy.get('body').then(($body) => {
     if ($body.text().includes('Delete')) {
       cy.get('[width="30"] > .checkbox-outer-container.check').click();
       cy.get('.btn').contains('Delete').click({ctrlKey: true});
       cy.get('.btn', { timeout: 20000 }).contains('Delete').should('not.exist');
-      cy.contains('No repositories have been added', { timeout: 20000 }).should('be.visible')
+      if (fleetCheck = true) {
```

Comment:
please add triple `===`. It seems to skip the check on the next `else`

Reply:
Ack, I will update it.